insights/tests/datasources/test_ls.py

```diff
@@ -59,8 +71,12 @@ def test_la_filterd():
 
 
 def test_lan():
-    ret = list_with_lan({})
-    assert ret == '/ /boot'
+    fstab = FSTab(context_wrap(FSTAB_CONTEXT))
+    broker = {
+        FSTab: fstab
+    }
```

Comment:
Better keep the original `test_lan` case with empty broker. 
```
def test_lan():
    ret = list_with_lan({})
    assert ret == '/ /boot'
```
And add this with `FSTab` one as a new test case.